taskcluster/rb_taskgraph/transforms/build_apk.py

```diff
@@ -16,6 +16,22 @@
 transforms = TransformSequence()
 
 
+@transforms.add
+def add_shippable_secrets(config, tasks):
+    for task in tasks:
+        secrets = task["run"].setdefault("secrets", [])
+
+        if task.pop("include-shippable-secrets", False) and config.params["level"] == "3":
+            secrets.extend([{
+                "key": key,
+                "name": 'project/mobile/reference-browser',
+                "path": target_file,
+            } for key, target_file in (
+                ('firebase', 'app/src/main/res/values/firebase.xml'),
+            )])
```

Comment:
Great idea for reusing `add_shippable_secrets()`, I hadn't thought of it 😃 Let's seize this opportunity to move the sentry token  here too ([Fenix does it too](https://github.com/mozilla-mobile/fenix/blob/ddd484387df450c2bfd43b3b0795f42109d7cc93/taskcluster/fenix_taskgraph/transforms/build.py#L49)).

Let's also move this function to the end of this file:
https://github.com/mozilla-mobile/reference-browser/blob/6cac15331f24947abf7e3c9e6ead763a97d500b8/taskcluster/rb_taskgraph/transforms/variant.py#L39

This way, both `build/kind.yml` and `build-bundle.yml` will benefit from it. As you can see they both use this transform:

https://github.com/mozilla-mobile/reference-browser/blob/6cac15331f24947abf7e3c9e6ead763a97d500b8/taskcluster/ci/build-bundle/kind.yml#L8

https://github.com/mozilla-mobile/reference-browser/blob/6cac15331f24947abf7e3c9e6ead763a97d500b8/taskcluster/ci/build/kind.yml#L8

Speaking of `variant.py`, we may eventually want to rename it to something else. Although, it's also used by `test/kind.yml`, and I think we should revisit that. That said, it's good for a longer term followup.

To sum up - short term: let's make `add_shippable_secrets()` handle sentry and move this function in `variant.py`.

Reply:
Done!